src/libse/Common/ArrayExtensions.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Nikse.SubtitleEdit.Core.Common
+{
+    public static class ArrayExtensions
```

Comment:
Maybe rename this to collection/list extension as all the method are for List<t>

Reply:
Good point! Renamed the class and file.